channels_redis/core.py

```diff
@@ -316,12 +317,59 @@ def _setup_encryption(self, symmetric_encryption_keys):
                 x.decode("utf8") for x in
                 await connection.zrange(key, 0, -1)
             ]
-        # TODO: More efficient implementation (lua script per shard?)
+
+        connection_to_channels, channel_to_message = (
+            self._map_channel_to_connection(channel_names, message)
+        )
```

Comment:
This is less than 120 characters, no need to wrap it

Reply:
will do